src/main/java/com/codecool/expertsystem/questionnaire/Answer.java

```diff
@@ -11,21 +10,26 @@
     private List<Value> valuesList = new ArrayList<>();
 
     public boolean evaluateAnswerByInput(String input) {
+
         List<String> values = new ArrayList<>();
```

Comment:
=newArrayList<> this is unnecessary

Reply:
it is neccessary, trust me.

src/main/java/com/codecool/expertsystem/questionnaire/Answer.java:[31,9] cannot find symbol
  symbol:   variable valuesList
  location: class com.codecool.expertsystem.questionnaire.Answer